src/DefaultBuilder/test/Microsoft.AspNetCore.Tests/Microsoft.AspNetCore.Tests.csproj

```diff
@@ -7,7 +7,7 @@
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore" />
     <Reference Include="Microsoft.AspNetCore.TestHost" />
-    <Content Include="Microsoft.AspNetCore.TestHost.StaticWebAssets.xml" CopyToOutputDirectory="Always" />
+    <Content Include="Microsoft.AspNetCore.TestHost.StaticWebAssets.xml" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Having `Always` means this project (and anything that depends on it) will *always* rebuild in VS.

Reply:
When you change branches, git will write the new source file, and it's timestamp will be newer, so the next build should still copy it.

Reply:
I would be very interested in knowing of a legitimate use case for `Always` as I'm not aware of one. I occasionally wonder if we could just treat it as an alias for `PreserveNewest`. `Always` is a foot gun, and we make it too easy for people to select it.

Reply:
@rainersigwald pointed to https://github.com/dotnet/msbuild/blob/14efa06d736aabb3af01ff827fff9426518f5fac/src/Tasks/Microsoft.Common.CurrentVersion.targets#L745-L748.  They default to `Never`, so these can't be removed.

Reply:
Agreed. However, we override that default in CSharp.Common.props for all C# test projects.

That said, I thought we did that for all projects. As-is, the override just creates inconsistencies. We should probably remove it at some point and _add_ missing `CopyToOutputDirectory` and `CopyToPublishDirectory` attributes. Something for another day&hellip;